### DISCUSSION TITLE
License locked (revalidating), no email received

### DISCUSSION THREAD

**Discussion by spamurai2000** [aclap-dev/video-downloadhelper#490] [Help and Q&A]
### Your question

My license is locked, have tried several times to "recheck" but never receive the email.  Have checked my spam folder.  Note my email address has a "+" as in <name+othername@domain.tld> and something in the email sending routine might not like this.  Have confirm the address works by sending from a 3rd email account.  Not putting the real address here for obvious reasons.

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.9 (stable)
Browser locale: en-US
Production build
Built on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google
Platform Win x86-32
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
Found companion app: VdhCoApp 2.0.19
Companion app binary: C:\Program Files (x86)\DownloadHelper CoApp\vdhcoapp.exe

DM me for serial and key if needed

**Comment by jcfrog** [comment]
hi,
please send me your email address at licenses@downloadhelper.net, I'll have a look.
